examples/single-page-app/myhub/myhub.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
```

Comment:
Question on the image assets above, what is their license?

Reply:
ai - generated - grey area - theoreticall my copyright - which i just donated